src/BottomUp.c

```diff
@@ -415,17 +521,16 @@ int parallel_bottomup_enqueue(QPTPool_t *pool,
     struct UserArgs *ua = NULL;
     QPTPool_get_args(pool, (void **) &ua);
 
-    struct BottomUp *root = (struct BottomUp *) malloc(ua->user_struct_size);
-    root->name_len = SNFORMAT_S(root->name, MAXPATH, 1, path, len);
+    struct BottomUp *root = (struct BottomUp *) calloc(ua->user_struct_size, 1);
+    new_pathname(root, path, len, NULL, 0);
 
     if (ua->generate_alt_name) {
         size_t name_len = root->name_len;
```

Comment:
Don't need this variable any more

Reply:
I ended up handling this a different way and changed `new_alt_pathname()` to return an error when the modified pathname is too big